Apps/W1/Shopify/app/src/Invoicing/Codeunits/ShpfyDraftOrdersAPI.Codeunit.al

```diff
@@ -0,0 +1,381 @@
+namespace Microsoft.Integration.Shopify;
+
+using Microsoft.Sales.Comment;
+using Microsoft.Sales.History;
+
+/// <summary>
+/// Codeunit Draft Orders API (ID 30159).
+/// </summary>
+codeunit 30159 "Shpfy Draft Orders API"
+{
+    Access = Internal;
+
+    var
+        ShpfyShop: Record "Shpfy Shop";
+        ShpfyCommunicationMgt: Codeunit "Shpfy Communication Mgt.";
+        ShpfyJsonHelper: Codeunit "Shpfy Json Helper";
+
+    internal procedure ExportOrderToShopify(
+        var TempShpfyOrderHeader: Record "Shpfy Order Header" temporary;
+        var TempShpfyOrderLine: Record "Shpfy Order Line" temporary;
+        var ShpfyOrderTaxLines: Dictionary of [Text, Decimal]
+    ) ResponseJsonToken: JsonToken
+    var
+        ShpfyBackgroundSyncs: Codeunit "Shpfy Background Syncs";
+        DraftOrderId: BigInteger;
+        NumberOfLines: Integer;
+        GraphQuery: TextBuilder;
+    begin
+        CreateDraftOrderGraphQL(TempShpfyOrderHeader, TempShpfyOrderLine, ShpfyOrderTaxLines, GraphQuery, NumberOfLines);
```

Comment:
Nit: I think it's cleaner if the CreateDraftOrderGraphQL would return a Text, instead of passing a TextBuilder by Reference.
Makes it cleaner from the "what are inputs/what are outputs" perspective.